frontend/lib/api/form/pollingstation/pollingStationUtils.test.ts

```diff
@@ -0,0 +1,417 @@
+  PollingStationValues,
+  resetFormSectionState,
+  ValidationResult,
+} from "@kiesraad/api";
+
+const defaultFormState: FormState = {
+  active: "recounted",
+  current: "recounted",
+  sections: {
+    recounted: {
+      index: 0,
+      id: "recounted",
+      isSaved: false,
+      ignoreWarnings: false,
+      errors: [],
+      warnings: [],
+    },
+    voters_votes_counts: {
+      index: 1,
+      id: "voters_votes_counts",
+      isSaved: false,
+      ignoreWarnings: false,
+      errors: [],
+      warnings: [],
+    },
+    differences_counts: {
+      index: 2,
+      id: "differences_counts",
+      isSaved: false,
+      ignoreWarnings: false,
+      errors: [],
+      warnings: [],
+    },
+    political_group_votes_1: {
+      index: 3,
+      id: "political_group_votes_1",
+      isSaved: false,
+      ignoreWarnings: false,
+      errors: [],
+      warnings: [],
+    },
+    political_group_votes_2: {
+      index: 4,
+      id: "political_group_votes_2",
+      isSaved: false,
+      ignoreWarnings: false,
+      errors: [],
+      warnings: [],
+    },
+    save: {
+      index: 5,
+      id: "save",
+      isSaved: false,
+      ignoreWarnings: false,
+      errors: [],
+      warnings: [],
+    },
+  },
+  unknown: {
+    errors: [],
+    warnings: [],
+  },
+  isCompleted: false,
+};
+
+const defaultValues: PollingStationValues = {
+  recounted: undefined,
+  voters_counts: {
+    poll_card_count: 0,
+    proxy_certificate_count: 0,
+    voter_card_count: 0,
+    total_admitted_voters_count: 0,
+  },
+  votes_counts: {
+    votes_candidates_counts: 0,
+    blank_votes_count: 0,
+    invalid_votes_count: 0,
+    total_votes_cast_count: 0,
+  },
+  voters_recounts: undefined,
+  differences_counts: {
+    more_ballots_count: 0,
+    fewer_ballots_count: 0,
+    unreturned_ballots_count: 0,
+    too_few_ballots_handed_out_count: 0,
+    too_many_ballots_handed_out_count: 0,
+    other_explanation_count: 0,
+    no_explanation_count: 0,
+  },
+  political_group_votes: [
+    {
+      number: 1,
+      total: 0,
+      candidate_votes: [
+        {
+          number: 1,
+          votes: 0,
+        },
+      ],
+    },
+    {
+      number: 2,
+      total: 0,
+      candidate_votes: [
+        {
+          number: 1,
+          votes: 0,
+        },
+      ],
+    },
+  ],
+};
+
+describe("PollingStationUtils", () => {
+  test("addValidationResultToFormState adds result to correct section", () => {
+    const formState = structuredClone(defaultFormState);
+    formState.sections.differences_counts.isSaved = true;
+    const validationResults: ValidationResult[] = [
+      {
+        fields: ["data.differences_counts.fewer_ballots_count"],
+        code: "F303",
+      },
+    ];
+
+    addValidationResultToFormState(formState, validationResults, "errors");
+
+    expect(formState.sections.differences_counts.errors.length).toBe(1);
+  });
+
+  // FIXME: F.204 is a global error, so is this test relevant?
```

Comment:
This should be addressed.

Reply:
this is a test technically for a global error (or specifically errors that span multiple sections)

Reply:
Ok then it's best to remove the FIXME comment :+1:

Reply:
I removed it... right, at least I cant find it 😅 ? also slightly tweaked the test